### DISCUSSION TITLE
Renewal page says student needs a university provided email to renew the pack

### DISCUSSION THREAD

**Discussion by sharadcodes** [community/community#61146] [GitHub Education]
### Hi everyone,

Our students do not have a University-provided email address to use for verification when trying to renew their GitHub Education pack.

Students came across message stating that they are required to provide University provided email to renew their pack.

We kindly request your assistance in resolving this matter. Is there a way for students without University-provided emails to still be eligible for the GitHub Education pack renewal?

Thank you

**Comment by samus-aran** [comment]
Hi @sharadcodes  👋

If you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "Other". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

It would also be recommended that either you or a faculty member provides evidence that students do not receive academic emails to help with the request. 👍

**Reply by Dilesha27** [reply]
Hi @sharadcodes,
         I already have student developer pack. but, I  got this mail. I attached that here. would you know how to apply student developer pack again?
![Screenshot (23)](https://github.com/community/community/assets/88300193/592788b8-bb96-40ee-bc34-678885dfc3bb)

**Comment by sharadcodes** [comment]
@samus-aran My ticket was Ticket ID: 2256417 and still no one has replied, its been over a week
